GeneratedModels/MSGraphGiphyRatingType.h

```diff
@@ -6,16 +6,16 @@
 
 typedef NS_ENUM(NSInteger, MSGraphGiphyRatingTypeValue){
 
-	MSGraphGiphyRatingTypeModerate = 0,
-	MSGraphGiphyRatingTypeStrict = 1,
+	MSGraphGiphyRatingTypeStrict = 0,

```

Comment:
This is a binary breaking change.

Reply:
Note that the new code is correct according to metadata:
```xml
<EnumType Name="giphyRatingType">
  <Member Name="strict" Value="0"/>
  <Member Name="moderate" Value="1"/>
  <Member Name="unknownFutureValue" Value="2"/>
</EnumType>
```

Reply:
Thanks for catching this. We also have the rename of ParticipantInfo as well that should be considered breaking.

Reply:
Yeah, the service changed underneath us. We kind of have to treat this as a bug fix, with the previous version being wrong. All of this is null since I doubt that released the original form. Ages like 🍷

Reply:
Aged like cheap wine. It was released.

Reply:
Given that the names stay as is and only values change for the enum, we should be safe under these conditions:
1. If our library is consumed as source code, i.e. gets recompiled with the application code. I am not that familiar with ObjC ecosystem.
2. Application code doesn't treat values as integers, e.g. comparison operation like `rating < moderate` or `rating == 1`

As you noted `ParticipantInfo` is even source breaking.